.github/workflows/release-code-freeze.yml

```diff
@@ -77,17 +91,37 @@ jobs:
         needs: code-freeze-prep
         if: ${{ needs.code-freeze-prep.outputs.freeze == 'true' && inputs.skipSlackPing != true }}
         steps:
-            - name: Slack
-              uses: archive/github-actions-slack@v2.0.0
-              id: notify
+            - name: Checkout code
```

Comment:
Since this job used to be just calling an action, no checkout/build was needed. this job is isolated from the others so needs its own setup now. In future we can either merge this back into the code freeze job or a lot of duplicate checkout and build won't be needed if we publish the CLI as a package.